node_modules/oae-core/manageaccess/js/manageaccess.js

```diff
@@ -175,7 +179,12 @@ define(['jquery', 'oae.core', 'jquery.autosuggest'], function($, oae) {
             infinityScroll.removeItems(principalId);
 
             // Mark the member as deleted
-            membersUpdates[principalId] = false;
+            if (pendingMembers[principalId]) {
+                delete membersUpdates[principalId];
+                pendingMembers[principalId] = false;
+            } else {
+                membersUpdates[principalId] = false;
+            }
```

Comment:
No I don't think it will work the same because if we updated a role of an existing user then decided to remove them after, it wouldn't perform the remove.